cmd/process-agent/info.go

```diff
@@ -162,25 +166,29 @@ func updateProcContainerCount(msgs []model.MessageBody) {
 }
 
 type queueStats struct {
-	processQueueSize    int
-	rtProcessQueueSize  int
-	eventQueueSize      int
-	podQueueSize        int
-	processQueueBytes   int64
-	rtProcessQueueBytes int64
-	eventQueueBytes     int64
-	podQueueBytes       int64
+	processQueueSize      int
+	rtProcessQueueSize    int
+	connectionsQueueSize  int
+	eventQueueSize        int
+	podQueueSize          int
+	processQueueBytes     int64
+	rtProcessQueueBytes   int64
+	connectionsQueueBytes int64
+	eventQueueBytes       int64
+	podQueueBytes         int64
 }
 
 func updateQueueStats(stats *queueStats) {
 	infoMutex.Lock()
 	defer infoMutex.Unlock()
```

Comment:
Do we actually need this mutex? I could see a case where the length and memory footprint of a queue don't correlate if the reads for these values are threaded in with these writes, how much do we care if that happens?

Reply:
Discussed offline, will bulk change in a subsequent PR.